examples/auth-methods/approle/README.md

```diff
@@ -0,0 +1,21 @@
+# AppRole Authentication
```

Comment:
This is more of a generic comment. I'm thinking about this now, w/ this repo structure. It may be frustrating to go into the docs only to realize that your language doesn't have examples. Is this still in the main ReadMe that we only have dotnet and go?

Reply:
Good point. I updated the readme, does that seem like it will suffice?

Reply:
yeah! Lgtm